### PR TITLE
refactor: clean and fix custom checks and use set_values for param_set

### PR DESCRIPTION
**Pull request by m-muecke**

### CODE REVIEW

File: R/LearnerClustAffinityPropagation.R
Comment (damirpolat) [inline R/LearnerClustAffinityPropagation.R:1]: Why do we need `crate`?

**Comment by damirpolat** [comment]
Thank you for the PR! I made a comment. Also, tests do not pass with the proposed changes.

File: R/LearnerClustAffinityPropagation.R
Comment (m-muecke) [reply R/LearnerClustAffinityPropagation.R:1]: just to leanify the environment, added recently in mlr3misc https://mlr3misc.mlr-org.com/reference/crate.html

**Comment by damirpolat** [comment]
Thank you, merging now!

**Pull request merged**